db/patch7.sql

```diff
@@ -0,0 +1,49 @@
+-- You should not modify if this have pushed to Github, unless it does serious wrong with the db.
+PRAGMA foreign_keys = off;
+
+BEGIN TRANSACTION;
+
+create table heartbeat_dg_tmp
+(
+    id INTEGER not null primary key autoincrement,
+    important INTEGER default 0 not null,
+    monitor_id INTEGER default 0 not null,
+    status INTEGER default 0 not null,
+    msg TEXT,
+    time DATETIME not null,
+    ping INTEGER default 0,
```

Comment:
ping was not default 0 before, did you change it on purpose?

Reply:
No, I didn't. What should it be?